dev-libs/glib/glib-2.77.1.ebuild

```diff
@@ -5,7 +5,9 @@ EAPI=8
 PYTHON_REQ_USE="xml(+)"
 PYTHON_COMPAT=( python3_{10..11} )
 
-inherit gnome.org gnome2-utils linux-info meson-multilib multilib python-any-r1 toolchain-funcs xdg
+inherit gnome.org gnome2-utils linux-info meson-multilib multilib plocale python-any-r1 toolchain-funcs xdg
+
+PLOCALES="ab af am an ar as ast az be be@latin bg bn bn_IN bs ca ca@valencia cs cy da de dz el en@shaw en_CA en_GB eo es et eu fa fi fr fur ga gd gl gu he hi hr hu hy id ie is it ja ka kk kn ko ku lt lv mai mg mk ml mn mr ms nb nds ne nl nn oc or pa pl ps pt pt_BR ro ru rw si sk sl sq sr sr@ije sr@latin sv ta te tg th tl tr tt ug uk vi wa xh yi zh_CN zh_HK zh_TW"
```

Comment:
What's the gain? Besides adding a huge maintenance cost of having to maintain this humongous list?

Reply:
Back when we removed the old approach for LINGUAS, we've decided that we won't be hacking around packages that don't naturally support it. Plocales was only tolerated because it was supposed only to be used when stripping away stuff had significant benefit. The proliferation of the eclass sounds like QA should have blocked it after all.